pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -102,6 +106,10 @@ message PreStartContainerResponse {
 // - Allocate allows Device Plugin to run device specific operations on
 //   the Devices requested
 message AllocateRequest {
```

Comment:
Since we are breaking the API should we call this PodAdmit ?

Reply:
Calling it PodAdmit may be a bit confusing since it is Kubelet side that does pod admission. On device plugin side, the operation is still mostly for device allocation.

Reply:
Ok lgtm